homeassistant/components/delijn/sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Support for De Lijn (Flemish public transport) information."""
+import logging
+
+from pydelijn.api import Passages
+import pytz
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, DEVICE_CLASS_TIMESTAMP)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = "Data provided by data.delijn.be"
+
+CONF_NEXT_DEPARTURE = 'next_departure'
+CONF_STOP_ID = 'stop_id'
+CONF_API_KEY = 'api_key'
+CONF_NUMBER_OF_DEPARTURES = 'number_of_departures'
+
+DEFAULT_NAME = 'De Lijn'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_NEXT_DEPARTURE): [{
+        vol.Required(CONF_STOP_ID): cv.string,
+        vol.Optional(CONF_NUMBER_OF_DEPARTURES, default=5): cv.positive_int}]
+})
+
+
+def due_in_minutes(timestamp):
```

Comment:
This isn't used.

Reply:
removed this unused leftover in latest commit